1-js/01-getting-started/3-code-editors/article.md

```diff
@@ -1,45 +1,61 @@
-# Code editors
+# Edytory kodu
 
-A code editor is the place where programmers spend most of their time.
+Edytor kodu to miejsce, w którym programista spędza najwięcej swojego czasu.
 
-There are two main types of code editors: IDEs and lightweight editors. Many people use one tool of each type.
+Wyróżniamy dwa typy edytorów: IDE oraz te "lekkie". Wielu ludzi korzysta z jednego narzędzia każdego typu.
 
 ## IDE
 
-The term [IDE](https://en.wikipedia.org/wiki/Integrated_development_environment) (Integrated Development Environment) refers to a powerful editor with many features that usually operates on a "whole project." As the name suggests, it's not just an editor, but a full-scale "development environment."
+Termin [IDE](https://pl.wikipedia.org/wiki/Zintegrowane_środowisko_programistyczne) (Zintegrowane Środowisko 
+Programistyczne) odnosi się do potężnego edytora z wieloma funkcjonalnościami, które współgrają z całym projektem
```

Comment:
maybe it will be better to remove ~~potężnego~~: `odnosi się do rozbudowanego edytora z wieloma funkcjonalnościami`